tests/upsert_validator/test_upsert_validator_vote.py

```diff
@@ -298,19 +299,22 @@ def test_get_validator_update(b, node_keys, node_key, ed25519_node_keys):
     assert not ValidatorElection.has_concluded(b, election.id, [tx_vote0, tx_vote1])
     assert ValidatorElection.has_concluded(b, election.id, [tx_vote0, tx_vote1, tx_vote2])
 
-    assert ValidatorElection.get_validator_update(b, 4, [tx_vote0]) == []
-    assert ValidatorElection.get_validator_update(b, 4, [tx_vote0, tx_vote1]) == []
+    assert not ValidatorElection.approved_update(b, 4, [tx_vote0])
+    assert not ValidatorElection.approved_update(b, 4, [tx_vote0, tx_vote1])
 
-    update = ValidatorElection.get_validator_update(b, 4, [tx_vote0, tx_vote1, tx_vote2])
-    update_public_key = codecs.encode(update[0].pub_key.data, 'base64').decode().rstrip('\n')
-    assert len(update) == 1
+    update = ValidatorElection.approved_update(b, 4, [tx_vote0, tx_vote1, tx_vote2])
+    update_public_key = None
+    if update:
+        update_public_key = codecs.encode(update.pub_key.data, 'base64').decode().rstrip('\n')
+    assert update
```

Comment:
We could do the following and avoid  `if` and `update_public_key = None`

```python
update = ValidatorElection.approved_update(b, 4, [tx_vote0, tx_vote1, tx_vote2]) 
assert update 
update_public_key = codecs.encode(update.pub_key.data, 'base64').decode().rstrip('\n') 
```

Reply:
Done